public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -166,8 +179,16 @@ <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Int
                 </div>
               </ng-template>
               <div class="tbl-r-group">
+                <div class="body-lead info-text tbl-d" *ngIf="checkCount(r.countValue); else Non20Counts;">
+                  &le; {{r.countValue | number}}
```

Comment:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=22&projectKey=DB&modal=detail&selectedIssue=DB-355

Converting this to number adds comma.